lib/msal-browser/docs/v4-migration.md

```diff
@@ -75,6 +75,17 @@ pca.handleRedirectPromise({
 })
 ```
 
+### Removal of some functions in `PublicClientApplication` 
+
+The following functions in `PublicClientApplication` have been removed:
+1. `enableAccountStorageEvents()` and `disableAccountStorageEvents()`: account storage events are now always enabled.
+1. `getAccountByHomeId()`, `getAccountByLocalId()`, and `getAccountByUsername()`: use `getAccount()` instead.
```

Comment:
This probably deserves its own section with before and after code snippets showing how to update from the previous usage to the new one